INFO  | 04/10/2020 03:50:14 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_transport_2020-Apr-10-15-50-14_11208\log_exp_11208.txt
INFO  | 04/10/2020 03:50:14 PM | File run_exp.py           | Line 27    | Exp instance id = 11208
INFO  | 04/10/2020 03:50:14 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 04/10/2020 03:50:14 PM | File run_exp.py           | Line 29    | Config =
INFO  | 04/10/2020 03:50:14 PM | File gran_runner.py       | Line 137   | Train/val/test = 8/2/2
INFO  | 04/10/2020 03:50:14 PM | File gran_runner.py       | Line 150   | No Edges vs. Edges in training set = 2.0
ERROR | 04/10/2020 03:50:30 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 38, in main
    runner.train()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 238, in train
    data = train_iterator.next()
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\dataloader.py", line 346, in __next__
    data = self.dataset_fetcher.fetch(index)  # may raise StopIteration
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\_utils\fetch.py", line 47, in fetch
    return self.collate_fn(data)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\dataset\gran_data.py", line 413, in collate_fn
    pos_list += np.concatenate((bb["positional1"][:, self.max_num_nodes - ii].reshape(-1, 1), np.zeros((2, ii))), axis=1)
ValueError: operands could not be broadcast together with shapes (0,) (2,11) 

